web-ui/custom-actions-for-alerts.markdown

```diff
@@ -104,8 +104,8 @@ When you get this to work as expected on the commmand line, you are ready to upl
 ## Example script: logging policy alert to syslog
 
 The following Custom action script will log the status and definition of a policy alert to syslog.
+
 ```bash
-[file=cfengine_custom_notification_policy_syslog.sh]
```

Comment:
This line is intentional / should be there to create the file components.
But looks like the site generation is buggy, so your commit does improve the end result. Let's merge it and I'll try to figure out what the real bug is here 😅

Reply:
Yep, this line was not the problem; https://github.com/cfengine/documentation/pull/3259